Formula/g/gyb.rb

```diff
@@ -20,15 +20,12 @@ class Gyb < Formula
 
   depends_on "python-setuptools" => :build
   depends_on "rust" => :build # for cryptography
+  depends_on "pyinstaller"
```

Comment:
should this be build dependency?

Reply:
Yes! In fact, because `pyinstaller` wraps everything into a frozen binary, _all_ of these dependencies can be build-only dependencies! 🎉

Reply:
👍